xyz

```diff
@@ -86,6 +90,7 @@ while (($# > 0)) ; do
     -b|--branch)    branch="$1"           ; shift ;;
     -i|--increment) increment="$1"        ; shift ;;
     -m|--message)   message_template="$1" ; shift ;;
+    -e|--edit)      commit_flags+=' -e'   ;;
```

Comment:
For lack of a better order, let's preserve alphabetical order here and in the `--help` text.